File: Algorithms/Implementation/grading students.cpp
/*
*Contributer : github.com/tanmayag8958
* Email : tanmayag8958@gmail.com
*/

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
     
    int n,x;
    cin>>n;
    int a[n];
    for(int i=0;i<n;i++)
        {
        cin>>a[i];
    }
    for(int j=0;j<n;j++)
        {
        if(a[j]%5>=3)
            {
            x=a[j]+(5-(a[j]%5));
            if(x<40)
                cout<<a[j]<<endl;
            else
                cout<<x<<endl;
        }
        else cout<<a[j]<<endl;
    }
    return 0;
}

File: Algorithms/Implementation/Grading Students.java
/*
* Contributer : github.com/shikhar-07
* Email : shikharanandg@gmail.com
*/
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class GradingStudents {
    
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int arr[] = new int[n];
        for(int i = 0; i < n; i++){
            int grade = in.nextInt();
                if(grade < 38){
                    arr[i] = grade;
                }else{
                    int m = grade + 1;
                    int s = grade + 2;
                    if(m%5==0){
                        arr[i] = m;
                    }else if(s%5==0){
                        arr[i] = s;
                    }else{
                        arr[i] = grade;
                    }
                }
        }
        for(int i=0;i<n;i++){
            System.out.println(arr[i]);
        }
    }
    
}
